### DISCUSSION TITLE
Option to have several windows stacked in an area and choose which is in front

### DISCUSSION THREAD

**Discussion by wetendorff** [rxhanson/RectanglePro-Community#280] [Ideas]
I have a super ultrawide monitor where I usually split the screen into three areas of equal size.

In the middle area, I have vscode. In the right area, I have Firefox development edition. And in the left area, I have Firefox, slack, email, VM, etc., all on top of each other. I can then press on the proper application in the dock so that the window moves to the front, but it could be more user-friendly.

It would be super helpful to show a mini dock that appears when the user hovers their mouse over a stacked area and simultaneously presses a modifier key. This mini dock would allow users to switch between stacked windows in that area, improving navigation and providing a more user-friendly experience.

This is how I do it now:

![Screen Recording](https://github.com/rxhanson/RectanglePro-Community/assets/168552/d1cd6f2c-62bc-4b8e-a7be-efee6b6b00c7)

**Comment by rxhanson** [comment]
Thanks for the feedback. I do have a feature planned for an additional dock that shows a line of windows associated with apps, but I haven't quite decided where that feature fits in (or when I want to prioritize it).  The feature kind of blurs the lines between something I would add into [Charmstone](https://charmstone.app) vs something I would add into Rectangle Pro, and I'll have to give it some more thought.
